records/generator.go

```diff
@@ -33,9 +33,36 @@ type RecordGenerator struct {
 	As         rrs
 	SRVs       rrs
 	SlaveIPs   map[string]string
+	EnumData   EnumerationData
 	httpClient http.Client
 }
 
+// EnumerableRecord is the lowest level object, and should map 1:1 with DNS records
```

Comment:
wondering if we should start an `api` package for types like this that we expose?